pyproject.toml

```diff
@@ -0,0 +1,134 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "spurt"
+description = "Spatial and Temporal phase Unwrapping for InSAR time-series"
+readme = "README.md"
+keywords = [
+  "insar",
+  "phase-unwrapping",
+  "radar",
+  "remote-sensing",
+  "sar",
+  "synthetic-aperture-radar",
+]
+classifiers = [
+  "Development Status :: 2 - Pre-Alpha",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: Apache Software License",
+  "License :: OSI Approved :: BSD License",
+  "License :: Other/Proprietary License",
+  "Operating System :: MacOS",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: Unix",
+  "Programming Language :: Python",
+  "Topic :: Scientific/Engineering",
+]
+requires-python = ">=3.9"
+dependencies = [
+  "numpy>=1.23",
+  "scipy>=1.9",
+  "ortools==9.8.3296",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+raster = [
+  "rasterio>=1.2",
+  "pyogrio>=0.7"
+]
+test = [
+  "pytest>=6",
+  "pytest-cov>=3"
+]
+
+[project.urls]
+Discussions = "https://github.com/isce-framework/spurt/discussions"
+Homepage = "https://github.com/isce-framework/spurt"
+Issues = "https://github.com/isce-framework/spurt/issues"
+
+[tool.black]
+preview = true
+
+[tool.mypy]
+enable_error_code = ["ignore-without-code", "redundant-expr", "truthy-bool"]
+strict = true
```

Comment:
Fair warning -- if you haven't used [Mypy](https://mypy-lang.org/) before, you might find that it takes some getting used to. It's a really nice tool, though, once you get the hang of it and understand its quirks.

If you find yourself struggling with Mypy errors later on, I'd recommend disabling strict mode here.

Reply:
Hmmm ... Let's start with this setting. Like you say, if we start struggling too much - will set it to False.